### PR TITLE
Riyuan: Structured the Website layout

### PR DESCRIPTION
**Pull request by Riyuanliu**
Created section for Home, About, Timeline, Project and contact
Created a Footer with social link

L=ENG-119
L=ENG-76
image
image

### CODE REVIEW

File: js/src/personal/riyuan/components/navigation/Navigation.tsx [hunk 1]
```diff
@@ -55,16 +58,16 @@ const NavItem = ({
   isOpen: boolean
 }) => (
   <div className={styles.navLink}>
-    <Link
-      to={`#${title}`}
+    <a
```
Comment (HaoxinLuo) [inline js/src/personal/riyuan/components/navigation/Navigation.tsx:61]: curious, what the difference is between Link, a, and Anchor that made you pick <a/> here?

File: js/src/personal/riyuan/components/navigation/Navigation.module.css [hunk 2]
```diff
@@ -40,6 +40,12 @@
...
 
+.header{
+  padding-top: 5px;
+  display: flex;
+}
+
+
 /************************************   */
 /* item list */
 /************************************   */
```
Comment (HaoxinLuo) [inline js/src/personal/riyuan/components/navigation/Navigation.module.css:51]: nit: feel free to do late, but once these sections or this file gets large, you should split it into even smaller css modules

File: js/src/personal/riyuan/components/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+const links = [
+  { link: '#', label: 'Contact' },
+  { link: '#', label: 'Privacy' },
+  { link: '#', label: 'Blog' },
+  { link: '#', label: 'Store' },
+  { link: '#', label: 'Careers' },
+]
+
+export function FooterCentered() {
+  const items = links.map((link) => (
```
Comment (HaoxinLuo) [inline js/src/personal/riyuan/components/footer/Footer.tsx:19]: nit: `items` and `links` are a bit vague especially when there are also hard coded external links to github and such later on.

**Review by HaoxinLuo — APPROVED** [review]

File: js/src/personal/riyuan/components/navigation/Navigation.tsx (hunk 1 above)
Comment (Riyuanliu) [reply js/src/personal/riyuan/components/navigation/Navigation.tsx:61]: I change it back to a tag, cause the link isnt working

File: js/src/personal/riyuan/components/footer/Footer.tsx (hunk 3 above)
Comment (Riyuanliu) [reply js/src/personal/riyuan/components/footer/Footer.tsx:19]: change items to social

**Pull request merged**